concrete/routes/api/account.php

```diff
@@ -0,0 +1,31 @@
+<?php
+defined('C5_EXECUTE') or die("Access Denied.");
+
+/**
+ * @var $router \Concrete\Core\Routing\Router
+ * @var $app \Concrete\Core\Application\Application
+ */
+
+use Concrete\Core\System\Info;
+
+$router->all('/account/info', function() use ($app) {
+    $request = $app->make(\Concrete\Core\Http\Request::class);
+    $loggedInUser = $request->attributes->get('oauth_user_id');
+
+    if (!$loggedInUser) {
+        throw new \RuntimeException('Invalid user associated with request');
+    }
+
+    $userRepository = $app->make(\Concrete\Core\User\UserInfoRepository::class);
+    $user = $userRepository->getByID($loggedInUser);
+
+    return new \League\Fractal\Resource\Item($user, function(\Concrete\Core\User\UserInfo $user) {
+        return [
+            'email' => $user->getUserEmail(),
+            'firstName' => $user->getAttributeValue('first_name'),
+            'lastName' => $user->getAttribute('last_name'),
```

Comment:
I'm trying to understand this bit. These attributes are not part of a blank installation, right?

Reply:
No, they are not included and so these values would be `null` on a blank install (which is what I used to develop this PR) They aren't actually used on the other side either, and this route deserves a controller that actually works well with scopes / OIDC